common/src/Microsoft.Azure.IIoT.Agent.Framework/src/Agent/Default/Worker.cs

```diff
@@ -282,7 +300,7 @@ public class Worker : IWorker, IDisposable {
         private class JobProcess : IDisposable {
 
             /// <inheritdoc/>
-            public JobProcessingInstructionModel JobContinuation { get; private set; }
+            public JobProcessingInstructionModel JobContinuation { get; set; }
```

Comment:
Is `JobContinuation` being set from outside ?

Reply:
nice catch, probably leftover from some fix attempt.